code/modules/mob/living/silicon/pai/software.dm

```diff
@@ -45,9 +45,9 @@
 	var/mob/living/silicon/pai/pai = user
 	data["available"] = available_software
 	data["records"] = list()
-	if("medical records" in pai.software)
+	if("Medical Records" in pai.software)
 		data["records"]["medical"] = medical_records
-	if("security records" in pai.software)
+	if("Security Records" in pai.software)
```

Comment:
use `PAI_PROGRAM_MEDICAL_RECORDS` instead of `"Medical Records"` and `PAI_PROGRAM_SECURITY_RECORDS` instead of `"Security Records"`

Reply:
Alright, changed em to use the defines